docs/config.toml

```diff
@@ -14,13 +14,14 @@ ui.diff-editor = "meld" # default, requires meld to be installed
 # ui.diff-editor = "vimdiff"
 
 ui.merge-editor = "meld" # default
+# ui.merge-editor = "vscode"
```

Comment:
Please squash into the parent commit

Reply:
Yes, I plan to. This will be a bit of a mess for a bit, I tried editing the PR using `github.dev`. Some of the new changes will be in a separate PR.